packages/core/test-project/ios/RNEmbraceTestProject.xcworkspace/contents.xcworkspacedata

```diff
@@ -1,9 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Workspace
    version = "1.0">
-   <FileRef
```

Comment:
was this intentional? Might be related to missing some of the files against that RNEmbraceTests target

Reply:
yes, this one it was intentionally. Some time ago I added by mistake this reference but it's not needed